lib/modules/manager/helm-values/types.ts

```diff
@@ -1,5 +1,14 @@
-export type HelmDockerImageDependency = {
+interface HelmDockerImageDependencyBasic {
```

Comment:
```suggestion
export interface HelmDockerImageDependencyBasic {
```

always export public types, even it it's only referenced indirectly